cpp/daal/src/algorithms/k_nearest_neighbors/oneapi/bf_knn_classification_predict_kernel_ucapi_impl.i

```diff
@@ -32,6 +33,9 @@
 
 #include "src/externals/service_ittnotify.h"
 
+const size_t maxInt32SizeT = static_cast<size_t>(daal::services::internal::MaxVal<int32_t>::get());
+const uint32_t maxInt32    = static_cast<uint32_t>(daal::services::internal::MaxVal<int32_t>::get());
```

Comment:
Inconsistency between var names
Could you replace const to constexpr?

Reply:
constexpr can't be compiled

Reply:
Var name was changed